packages/tasit-sdk/src/Decentraland.test.js

```diff
@@ -1,38 +1,36 @@
-import { Account, Action } from "./TasitSdk";
-const {
-  ConfigLoader,
-  ERC20,
-  ERC721,
-  Marketplace: MarketplaceContracts,
-} = Action;
+import { Account, Action, ContractBasedAccount } from "./TasitSdk";
+const { ConfigLoader, ERC20, ERC721, Marketplace: marketplaces } = Action;
```

Comment:
Because it's a shorter name than `MarketplaceContracts`

Reply:
Would `Marketplace` work though, without the renaming?

Reply:
Sure! I usually do that to avoid conflicts between that var and `TasitContracts.Marketplace` var. That isn't the case anymore (I've moved the addresses to `mochaSetup.js` file).

Reply:
Done.

Reply:
Makes sense. Thanks! Resolving